### PR TITLE
Feature/conect endpoints hf timeline

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/api/RestAPI.js [hunk 1]
```diff
@@ -225,16 +224,29 @@ class RestAPI {
   }
 
   /**
-   * Get the human footprint timeline data in the given area.
+   * Get the human footprint timeline data for a specific strategic ecosystem in the given area.
```
Comment (erikasv) [inline src/api/RestAPI.js:227]: This endpoitn brings the data for the geofence

File: src/landscape/humanFootprint/TimelineFootprint.jsx [hunk 2]
```diff
@@ -75,11 +75,18 @@ class TimelineFootprint extends React.Component {
...
+      RestAPI.requestSEHFTimeline(areaId, geofenceId, 'Páramo'),
+      RestAPI.requestSEHFTimeline(areaId, geofenceId, 'Humedal'),
+      RestAPI.requestSEHFTimeline(areaId, geofenceId, 'Bosque Seco Tropical'),
+      RestAPI.requestTotalHFTimeline(areaId, geofenceId),
+    ])
+      .then(([paramo, wetland, dryForest, aTotal]) => {
+        this.setState({ hfTimeline: this.processData([paramo, wetland, dryForest, aTotal]) });
       })
-      .catch(() => {});
+      .catch(() => this.reportConnError());
```
Comment (erikasv) [inline src/landscape/humanFootprint/TimelineFootprint.jsx:89]: this function doesn't exist in this component

File: src/api/RestAPI.js (hunk 1 above)
Comment (cazapatamar) [reply src/api/RestAPI.js]: Done

File: src/landscape/humanFootprint/TimelineFootprint.jsx (hunk 2 above)
Comment (cazapatamar) [reply src/landscape/humanFootprint/TimelineFootprint.jsx]: Done

**Pull request merged**